Tools/Scripts/run-jsc-stress-tests

```diff
@@ -186,6 +186,7 @@ def usage
     puts "--jsc                (-j)   Path to JavaScriptCore build product. This option is required."
     puts "--no-copy                   Do not copy the JavaScriptCore build product before testing."
     puts "                            --jsc specifies an already present JavaScriptCore to test."
+    puts "--platform                  Indicate the target platform we are running on."
```

Comment:
What does the arguments looks like? Maybe add some examples?

Reply:
Oh, I see they are in the `computeFullConfiguration` function.

Reply:
Yes, they are computed in `computeFullConfiguration`.